rllib/core/testing/utils.py

```diff
@@ -105,15 +105,20 @@ def get_trainer_runner(
     env: "gym.Env",
     scaling_config: TrainerScalingConfig,
     is_multi_agent: bool = False,
+    eager_tracing: bool = False,
```

Comment:
needs some arg doc to say that eager_tracing is only relevant in the tf case.

Reply:
kk